frontend/src/metabase/.eslintrc

```diff
@@ -32,7 +32,8 @@
     {
       "files": ["**/*.stories.tsx"],
       "rules": {
-        "import/no-default-export": 0
+        "import/no-default-export": 0,
+        "import/no-unresolved": [2, {"ignore": ["@storybook"]}]
```

Comment:
allow unresolved imports to `@storybook/something` in stories. Stories are located near components, but storybook is a part of different package